### ISSUE TITLE
One SPEC to cover both creating and using nightlies

### ISSUE THREAD

**Issue by betatim** [scientific-python/specs#188]
I'd like to propose having both "create nightlies" and "test with nightlies" in one SPEC (probably SPEC 4). That way we'd have both things related to nightlies in one place instead of spread over two SPECs. The other advantage is that I think we have pretty good agreement on the nightlies part, so we could get the SPEC done. I think the CI Best Practices will be a longer process because it touches more different topics.

WDYT?

cc @bsipocz @Carreau

**Comment by bsipocz** [comment]
Yes, sounds reasonable. Putting them in one place we can certainly wrap up SPEC4 this week, while I agree that spec5 may indeed drag on longer.

**Comment by Carreau** [comment]
I don't have any objection. 

I think one of the complexity with publishing nightly is managing tokens, but an idea I have is to have a virtual index on a server something that takes number of index-es, and make a single page. The `/simple/` format should be relatively easy to create using beautiful soup or libxml. 
This would let many packages publish on different places, without us having to take care of tokens, and/or  subproject to update their CI with --add-index.
